opencensus/log/__init__.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+from copy import copy
+import logging
+
+from opencensus.trace import execution_context
+
+
+_meta_logger = logging.getLogger(__name__)
+
+TRACE_ID_KEY = 'traceId'
+SPAN_ID_KEY = 'spanId'
+SAMPLING_DECISION_KEY = 'traceSampled'
+
+LogAttrs = namedtuple('LogAttrs', ['trace_id', 'span_id', 'sampling_decision'])
```

Comment:
Do we want to add `SpanContext.tracestate`? Tracestate is designed for interop with other tracing systems, do we expect extensions (or exporters) to extend the log_attrs?

For example, tracing system FOOBAR has a legacy correlation_id in `SpanContext.tracestate['correlation_id@foobar']`, the FOOBAR log exporter would expect to have that correlation_id associated with log entries.

Reply:
A similar question regarding tags, do we expect to put tags to the log entries in the future?

Reply:
I expect we will eventually, but we probably don't want to dump a lot of context into logs by default. In any case, not in this PR.